src/m365/spo/commands/roledefinition/roleType.ts

```diff
@@ -0,0 +1,11 @@
+export enum RoleType {
```

Comment:
Let's change the file name to `RoleType.ts` to make it clear that it's a type definition and not a module

Reply:
✅